src/main.ml

```diff
@@ -270,7 +274,9 @@ let parse_rules_from_stdin () =
   stdin |> Sexp.input_rev_sexps |> List.filter_map Rule.of_sexp
 
 let with_output f =
-  let json = open_out "compile_commands.json" in
+  let outfile_name = Filename.concat !output_directory "compile_commands.json" in
+  if !debug then Format.eprintf "Saving to file %s" outfile_name;
```

Comment:
A newline at the end here might be useful

Reply:
Example output:

```
file: lib/bb_blob_stubs.cpp, c_file: /dune_build/default/lib/bb_blob_stubs.cpp
Compilation database has 3 entries
Saving to file /build/compile_commands.json
```